packages/auth/test/integration/flows/totp.test.ts

```diff
@@ -46,112 +46,131 @@ describe(' Integration tests: Mfa TOTP', () => {
   let auth: Auth;
   let totpSecret: TotpSecret;
   let displayName: string;
+  let totpTimestamp: Date;
+  let emulatorUrl: string | null;
   beforeEach(async () => {
-    auth = getTestInstance();
-    displayName = 'totp-integration-test';
+    emulatorUrl = getEmulatorUrl();
+    if (!emulatorUrl) {
+      auth = getTestInstance();
+      displayName = 'totp-integration-test';
+    }
   });
 
   afterEach(async () => {
-    await cleanUpTestInstance(auth);
+    if (!emulatorUrl) {
+      await cleanUpTestInstance(auth);
+    }
   });
 
   it('should not enroll if incorrect totp supplied', async () => {
-    const cr = await signInWithEmailAndPassword(auth, email, 'password');
-    const mfaUser = multiFactor(cr.user);
-    const session = await mfaUser.getSession();
-    totpSecret = await TotpMultiFactorGenerator.generateSecret(session);
-    const multiFactorAssertion =
-      TotpMultiFactorGenerator.assertionForEnrollment(
-        totpSecret,
-        incorrectTotpCode
-      );
-
-    await expect(
-      mfaUser.enroll(multiFactorAssertion, displayName)
-    ).to.be.rejectedWith('auth/invalid-verification-code');
+    if (!emulatorUrl) {
```

Comment:
nit: Can we use the skip method instead? https://stackoverflow.com/a/32820119, that way it is obvious that the test is being skipped.

Reply:
ACK